src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ObjectionStatus.java

```diff
@@ -1,9 +1,15 @@
 package uk.gov.companieshouse.api.strikeoffobjections.model.entity;
 
 public enum ObjectionStatus {
+
+    INELIGIBLE_COMPANY_STRUCK_OFF,
+    INELIGIBLE_NO_DISSOLUTION_ACTION,
     OPEN,
     PROCESSED,
-    SUBMITTED,
-    INELIGIBLE_COMPANY_STRUCK_OFF,
-    INELIGIBLE_NO_DISSOLUTION_ACTION
+    SUBMITTED;
+
+    public boolean isIneligibleStatus() {
```

Comment:
How about just 'isIneligible()' for this method name? Status is already present in/clear from the class name.